framework/src/utils/JsonSyntaxTree.C

```diff
@@ -174,7 +175,11 @@ JsonSyntaxTree::addParameters(const std::string & parent,
     json["parent_syntax"] = parent;
     json["description"] = params->getClassDescription();
     if (lineinfo.isValid())
+    {
       json["file_info"][lineinfo.file()] = lineinfo.line();
```

Comment:
No, for actions it would use the branch above that uses `json[action]["file_info"]`.